.env.example

```diff
@@ -0,0 +1,46 @@
+# if you want to use localstack pro, else delete this env
+LOCALSTACK_IMAGE=localstack/localstack-pro:1.4.0
+LOCALSTACK_API_KEY=<your api key>
+
+POSTGRES_PASSWORD=oCLsSsvzUGq3w1QWqNKnJPyosyfcvfGZ
```

Comment:
did you upload the secrets to the example file by accident or on purpose?

Reply:
On purpose. These instances are only running locally or in the github action, so no harm done.
Sonar already did not like it.
Good catch. We could generate the files with terraform, but its not that important